src/App.js

```diff
@@ -1,13 +1,92 @@
-import React from 'react';
-import './App.css';
-
-function App() {
-  return (
-      <h1>Handbags & Purses</h1>
-  );
-}
-
-export default App;
-
-
-
+import React from 'react';
+import './App.css';
+import bag1 from "./assets/bag_1.png";
+import bag2 from "./assets/bag_2.png";
+import bag3 from "./assets/bag_3.png";
+import bag4 from "./assets/bag_4.png";
+import brand from "./assets/brand.png"
+import our_story from "./assets/our_story.png"
+import Nav from './Nav';
+import Product from './Product'
+import Tiles from './Tiles'
+function App() {
+  return ( <>
+        <header>

```

Comment:
In het hele stuk geld {} is voor variabelen "" is voor waardes. {""}  werkt scheinbaar wel maar is fout en kost punten.
dus
const content = "to the collection"; // hier stop ik het in een variabele
title = {content} hier geef ik de inhoud van de variabele content door aan title.
of
title = "to the collection" hier geef ik title direkt de waarde "to the collection"